modes/config-mode/.webpack/webpack.prod.js

```diff
@@ -43,6 +43,12 @@ const config = {
         amd: '@ohif/mode-longitudinal',
         root: '@ohif/mode-longitudinal',
       },
+      '@ohif/mode-tmtv': {
```

Comment:
Just a comment here: those webpack.prod.js file configuration are quite awkward. We are making a huge coupling

Reply:
would be nice if we just get OHIFs modes and use it

Reply:
They are horrible to use like this, but I don't see a good way of separating out the source code without making mroe changes to core OHIF, which I hope to do eventually, but need to get agreement from MGH that this is a good thing to do.